web/src/pages/api/v1/precheck/[app_id].ts

```diff
@@ -189,7 +205,28 @@ export default async function handlePrecheck(
       req
     );
   }
+  const app_metadata: AppMetadataPayload | undefined = Array.isArray(
```

Comment:
I may be wrong but I believe `rawAppValues.app_metadata` should always be an array?